certified-connectors/Litera/apiDefinition.swagger.json

```diff
@@ -0,0 +1,198 @@
+	"paths": {
+		"/GetMatterList": {
+			"get": {
+				"tags": [
+					"Matter List"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"produces": [
+					"application/json"
+				],
+				"summary": "Allows the user to search for legal matters and their associated IDs.",
+				"description": "Allows the user to search for legal matters (also known as deals or transactions) and their associated IDs. The endpoint returns a list of matter IDs and their corresponding matter names which satisfy the given natural language request. The request can contain search criteria involving matter fields such as matter location, industry, client name, etc. Example requests include \\\"Show me New York matters\\\", \\\"Get me matters for clients in the financial industry\\\", \\\"Find me deals in tech\\\".",
+				"operationId": "GetMatterList",
+				"parameters": [
+					{
+						"name": "request",
+						"description": "Natural language search request for legal clauses or definitions",
+						"in": "query",
+						"type": "string",
+						"x-ms-summary": "Natural language search request for legal clauses or definitions",
+						"required": true
+					}
+				]
+			}
+		},
+		"/GetMatterNarrative": {
+			"get": {
+				"tags": [
+					"Matter Narrative"
+				],
+				"produces": [
+					"application/json"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"parameters": [
+					{
+						"name": "matterId",
+						"description": "The matter ID (matter number) of the matter for which to retrieve the narrative.",
+						"in": "query",
+						"type": "string",
+						"required": true,
+						"x-ms-summary": "The matter ID (matter number) of the matter for which to retrieve the narrative."
+					}
+				],
+				"summary": "Allows the user to retrieve a matter narrative for the specified legal matter.",
+				"description": "Allows the user to retrieve a matter (aka deal, transaction, or case) narrative -- i.e. the descriptive overview -- for the specified matter. Use this function to handle ANY request for a matter narrative, overview, or description. The endpoint returns the narrative for the matter specified by matter name. A matter narrative is one or two paragraphs of prose text that gives a descriptive overview of the matter. Users will request a matter narrative when they want a quick overview of a legal matter. The user might not use the term narrative, but the function can be called whenever the user requests some kind of overview or general information about a specific matter.",
+				"operationId": "GetMatterNarrative"
+			}
+		},
+		"/GetMatterDetail": {
+			"get": {
+				"tags": [
+					"Matter Detail"
+				],
+				"produces": [
+					"application/json"
+				],
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {}
+					}
+				},
+				"summary": "Allows the user to retrieve details of a legal matter.",
```

Comment:
'summary' value must end with alphanumeric character. ValuePath: GetMatterDetail/get/summary

Reply:
done - will be reflected in next commit.